dbms/src/Interpreters/Settings.h

```diff
@@ -128,7 +128,8 @@ struct Settings
     M(SettingUInt64, log_queries_cut_to_length, 100000, "If query length is greater than specified threshold (in bytes), then cut query when writing to query log. Also limit length of "                                               \
                                                         "printed query in ordinary text log.")                                                                                                                                          \
                                                                                                                                                                                                                                         \
-    M(SettingUInt64, max_concurrent_queries_for_user, 0, "The maximum number of concurrent requests per user.")                                                                                                                         \
+    M(SettingMemoryLimit, max_concurrent_queries_for_user, UInt64(0), "Maximum memory usage for processing of single query. Can either be an UInt64 (means memory limit in bytes), "                                                    \
```

Comment:
Why change this? It is the maxmum number of concurrent request per user

Reply:
my fault, fixing

Reply:
fixed